wiki/Tournaments/YSC/2021/en.md

```diff
@@ -16,4 +58,693 @@ The **Yaz's Summer Cup 2021** (***YSC 2021***) is a 1v1 double-elimination osu!
+- Where rules do not prevail, common sense shall. The administration has explicit discretion to apply their judgement on this as they see fit.
+- Each stage will be held on a Saturday/Sunday. Schedules and mappools will be released on the Monday/Tuesday before that stage is due to happen, to allow for reschedules.
+- Reschedules will be allowed if both players agree on a time and provide proof of the agreed reschedule to hosts/co-hosts/scheduling team.
+- Please be aware that reschedules must be submitted before 23:59UTC Friday before a round is due to happen. Any reschedules later than this will only be accepted on a case by case basis in extenuating circumstances.
+- In order for your registration to count, you must fill out the form provided, join the Discord server and change your Discord username to your osu! username.
+
+### Qualifiers
+
+- Players will play the qualifiers mappool once.
+- Failed scores are counted as a pass.
+- The Top 64 qualifying scores will progress into the knockout stage.
+
+### Knockout stages
+
+- Bracket matchups will be assigned as #1 vs #64, #2 vs #63, etc.
+- Referees will create the lobby and invite players 10 minutes before the match is due to start.
+- Any player that fails to show up 10 minutes after a match is due to start will forfeit the match and a win by default will be awarded to the other player.
+- Failure to attend a match without a valid reason may result in being blacklisted from participating in future tournaments.
+- The referee will have host at all times during the match.
+- Players will have one ban in the Round of 64 and Round of 32 and two bans thereafter.
+- Players are allowed to ban from the same mappool.
+- Players will roll once. The winner of the roll selects first or second pick or ban, and the loser of the roll selects from the remaining options.
+- Players will have 90 seconds to ban a map. Failure to ban a map after this time will result in a loss of a ban.
+- Players will have 90 seconds to pick a map. Failure to pick a map after this time will result in the pick being passed to their opponent.
+- If a disconnection or a technical issue occurs within the first 25% of a map. The map will be allowed to be replayed. Each player has a limit of 1 replay per match, any disconnections or technical issues after a replay has been used will result in the point going over to the other player.
+- In the case of a disconnection, common sense will be applied: if proof exists of the disconnected player's score, it will be considered valid, and the winner of the point will be judged accordingly.
+- Any attempts to stall the match, show toxicity or spam during the match will result in a warning. Failure to comply with this rule again will result in a point being awarded to another player.
+- Failed scores are counted as a pass.
+- If there is a tie score on a map, the map will be nullified for the duration of the match and cannot be re-picked. This will not affect the pick order.
```

Comment:
```suggestion
- If there are tied scores on a map, the map will be nullified for the duration of the match and cannot be re-picked. This will not affect the pick order.
```